src/test/java/com/google/sps/MatchServletTest.java

```diff
@@ -0,0 +1,75 @@
+package com.google.sps;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+import com.google.gson.Gson;
+import com.google.sps.data.Match;
+import com.google.sps.data.MatchRepository;
+import com.google.sps.data.NonPersistentMatchRepository;
+import com.google.sps.data.User;
+import com.google.sps.servlets.MatchServlet;
+import java.io.*;
+import javax.servlet.http.*;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Collection;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class MatchServletTest {
+    @Mock
+    HttpServletRequest request;
+
+    @Mock
+    HttpServletResponse response;
+
+    private static MatchRepository testRepository;
+    private static User testUser;
+
+    @Before
+    public void setUp() throws Exception {
+        MockitoAnnotations.initMocks(this);
+
+        NonPersistentMatchRepository repository = new NonPersistentMatchRepository();
+        testUser = repository.addTestData();
+        testRepository = repository;
+    }
+
+    @Test
+    public void testMatchDoGet() throws IOException, ServletException {
```

Comment:
Consider changing the tests names to specify the condition being tested and expected outcome.

see:  
![image](https://user-images.githubusercontent.com/5056759/85776987-0fa5c480-b6d6-11ea-9021-1448d1e8dc5f.png)

Reply:
fixed this, thanks for the feedback